controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/ServiceRegistry.java

```diff
@@ -57,6 +58,8 @@ public interface ServiceRegistry {
 
     NameService nameService();
 
+    VpcEndpointService vpcEndpointService();
```

Comment:
Consider a more generic name. `PrivateEndpointService`?

Reply:
Isn't VPC "virtual private cloud", and that concept is used in other clouds too? 
It should in any case be a `...EndpointServiceService` >_<